test/resource/wasm/mul.b64

```diff
@@ -0,0 +1 @@
IAhFDQELIAggCjYCGAJAIAAoAhAiA0UNACAIIAM2AhAgAyAINgIYCyAAQRRqKAIAIgNFDQAgCEEUaiADNgIAIAMgCDYCGAsCQAJAIARBD0sNACAAIAQgAmoiA0EDcjYCBCAAIANqIgMgAygCBEEBcjYCBAwBCyAAIAJqIgUgBEEBcjYCBCAAIAJBA3I2AgQgBSAEaiAENgIAAkAgB0UNACAHQXhxQayAwIAAaiECQQAoApiAwIAAIQMCQAJAQQEgB0EDdnQiCCAGcQ0AQQAgCCAGcjYChIDAgAAgAiEIDAELIAIoAgghCAsgCCADNgIMIAIgAzYCCCADIAI2AgwgAyAINgIIC0EAIAU2ApiAwIAAQQAgBDYCjIDAgAALIABBCGohAwsgAUEQaiSAgICAACADCwoAIAAQhoCAgAAL4g0BB38CQCAARQ0AIABBeGoiASAAQXxqKAIAIgJBeHEiAGohAwJAIAJBAXENACACQQNxRQ0BIAEgASgCACICayIBQQAoApSAwIAAIgRJDQEgAiAAaiEAAkAgAUEAKAKYgMCAAEYNAAJAIAJB/wFLDQAgASgCCCIEIAJBA3YiBUEDdEGsgMCAAGoiBkYaAkAgASgCDCICIARHDQBBAEEAKAKEgMCAAEF+IAV3cTYChIDAgAAMAwsgAiAGRhogAiAENgIIIAQgAjYCDAwCCyABKAIYIQcCQAJAIAEoAgwiBiABRg0AIAEoAggiAiAESRogBiACNgIIIAIgBjYCDAwBCwJAIAFBFGoiAigCACIEDQAgAUEQaiICKAIAIgQNAEEAIQYMAQsDQCACIQUgBCIGQRRqIgIoAgAiBA0AIAZBEGohAiAGKAIQIgQNAAsgBUEANgIACyAHRQ0BAkACQCABIAEoAhwiBEECdEG0gsCAAGoiAigCAEcNACACIAY2AgAgBg0BQQBBACgCiIDAgABBfiAEd3E2AoiAwIAADAMLIAdBEEEUIAcoAhAgAUYbaiAGNgIAIAZFDQILIAYgBzYCGAJAIAEoAhAiAkUNACAGIAI2AhAgAiAGNgIYCyABKAIUIgJFDQEgBkEUaiACNgIAIAIgBjYCGAwBCyADKAIEIgJBA3FBA0cNACADIAJBfnE2AgRBACAANgKMgMCAACABIABqIAA2AgAgASAAQQFyNgIEDwsgASADTw0AIAMoAgQiAkEBcUUNAAJAAkAgAkECcQ0AAkAgA0EAKAKcgMCAAEcNAEEAIAE2ApyAwIAAQQBBACgCkIDAgAAgAGoiADYCkIDAgAAgASAAQQFyNgIEIAFBACgCmIDAgABHDQNBAEEANgKMgMCAAEEAQQA2ApiAwIAADwsCQCADQQAoApiAwIAARw0AQQAgATYCmIDAgABBAEEAKAKMgMCAACAAaiIANgKMgMCAACABIABBAXI2AgQgASAAaiAANgIADwsgAkF4cSAAaiEAAkACQCACQf8BSw0AIAMoAggiBCACQQN2IgVBA3RBrIDAgABqIgZGGgJAIAMoAgwiAiAERw0AQQBBACgChIDAgABBfiAFd3E2AoSAwIAADAILIAIgBkYaIAIgBDYCCCAEIAI2AgwMAQsgAygCGCEHAkACQCADKAIMIgYgA0YNACADKAIIIgJBACgClIDAgABJGiAGIAI2AgggAiAGNgIMDAELAkAgA0EUaiICKAIAIgQNACADQRBqIgIoAgAiBA0AQQAhBgwBCwNAIAIhBSAEIgZBFGoiAigCACIEDQAgBkEQaiECIAYoAhAiBA0ACyAFQQA2AgALIAdFDQACQAJAIAMgAygCHCIEQQJ0QbSCwIAAaiICKAIARw0AIAIgBjYCACAGDQFBAEEAKAKIgMCAAEF+IAR3cTYCiIDAgAAMAgsgB0EQQRQgBygCECADRhtqIAY2AgAgBkUNAQsgBiAHNgIYAkAgAygCECICRQ0AIAYgAjYCECACIAY2AhgLIAMoAhQiAkUNACAGQRRqIAI2AgAgAiAGNgIYCyABIABqIAA2AgAgASAAQQFyNgIEIAFBACgCmIDAgABHDQFBACAANgKMgMCAAA8LIAMgAkF+cTYCBCABIABqIAA2AgAgASAAQQFyNgIECwJAIABB/wFLDQAgAEF4cUGsgMCAAGohAgJAAkBBACgChIDAgAAiBEEBIABBA3Z0IgBxDQBBACAEIAByNgKEgMCAACACIQAMAQsgAigCCCEACyAAIAE2AgwgAiABNgIIIAEgAjYCDCABIAA2AggPC0EfIQICQCAAQf///wdLDQAgAEEIdiICIAJBgP4/akEQdkEIcSICdCIEIARBgOAfakEQdkEEcSIEdCIGIAZBgIAPakEQdkECcSIGdEEPdiACIARyIAZyayICQQF0IAAgAkEVanZBAXFyQRxqIQILIAEgAjYCHCABQgA3AhAgAkECdEG0gsCAAGohBAJAAkBBACgCiIDAgAAiBkEBIAJ0IgNxDQAgBCABNgIAQQAgBiADcjYCiIDAgAAgASAENgIYIAEgATYCCCABIAE2AgwMAQsgAEEAQRkgAkEBdmsgAkEfRht0IQIgBCgCACEGAkADQCAGIgQoAgRBeHEgAEYNASACQR12IQYgAkEBdCECIAQgBkEEcWpBEGoiAygCACIGDQALIAMgATYCACABIAQ2AhggASABNgIMIAEgATYCCAwBCyAEKAIIIgAgATYCDCAEIAE2AgggAUEANgIYIAEgBDYCDCABIAA2AggLQQBBACgCpIDAgABBf2oiAUF/IAEbNgKkgMCAAAsLBAAAAAtOAAJAIAANAD8AQRB0DwsCQCAAQf//A3ENACAAQX9MDQACQCAAQRB2QAAiAEF/Rw0AQQBBMDYC9IPAgABBfw8LIABBEHQPCxCHgICAAAALAgALDgAQiYCAgAAQiYCAgAALCgAgABCDgICAAAsKACAAEIWAgIAACwoAIAAgARAAEAoLCAAgABALEAoLCAAgABAMEAoLCw0BAEGAgMAACwQCAAAA
```

Comment:
Where did this deus-ex-machina file appear from?
I assume it is the result of a rust compilation of something, but what source file, how was the compilation done?
I think it's a big mistake to insert compilation results like this into our repository - we need to have the source code and the build script to generate them, in case a year from now somebody will need to modify these files.

Reply:
We are planning to add the build scripts for WASM tests when we start adding tests translated from Cassandra. Let's replace the built UDFs then.
I do agree that I should include the source alongside this, I'm adding it in a rebase.